app/assets/stylesheets/application.css.scss

```diff
@@ -25,26 +29,104 @@ body {
   background-size: cover;
 }
 
-#user_profile {
+footer {
+  background-color: $black;
+  height: auto;
+}
+
+.tattoo-thumbnail {
+  max-height: 150px;
+  max-width: 200px;
+}
+
+.banner-image {
+  padding-top: 15px;
+}
+
+footer {
+  background-color: $oil;
+  position: fixed;
```

Comment:
Properties should be ordered background-color, bottom, color, height, padding-top, position, text-align, width